visualvm/spoon.me

```diff
@@ -0,0 +1,69 @@
+#
+# Licensed under the Apache License, Version 2.0
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Setup meta
+meta title="visualvm"
+meta namespace="visualvm" 
+meta name="visualvm"
+
+#Basic dependencies and tools
+from base,oracle/jdk:8
```

Comment:
Because I want to see the JVM's running on the host. 

Never tried with clean...probably works fine.